modules/builds-identifying-image-change-triggers.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * builds/triggering-builds-build-hooks.adoc
+
+[id="builds-image-change-trigger-identification_{context}"]
+= Indentifying the image change trigger of a build
+
+As a developer, if you have image change triggers, you can identify which image change initiated the last build. This can be useful for debugging or troubleshooting builds.
+
+.Example `BuildConfig`
+[source,yaml]
+----
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+  metadata:
+    name: bc-ict-example
+    namespace: bc-ict-example-namespace
+  spec:
+
+    ...
+
+    triggers:
+    - imageChange:
+        from:
+          kind: ImageStreamTag
+          name: input:latest
+          namespace: bc-ict-example-namespace
+      type: ImageChange
+  status:
+    imageChangeTriggers:
+    - from:
+        name: input:latest
+        namespace: bc-ict-example-namespace
+      lastTriggerTime: "2021-06-30T13:47:53Z"
+      lastTriggeredImageID: image-registry.openshift-image-registry.svc:5000/bc-ict-example-namespace/input@sha256:0f88ffbeb9d25525720bfa3524cb1bf0908b7f791057cf1acfae917b11266a69
```

Comment:
And add after this line:

```
- from:
     name: input2:latest
     namespace: bc-ict-example-namespace
   lastTriggeredImageID:  image-registry.openshift-image-registry.svc:5000/bc-ict-example-namespace/input2@sha256:0f88ffbeb9d25525720bfa3524cb2ce0908b7f791057cf1acfae917b11266a69
```

note, I omitted the time with input2 since it was not the one that was used to trigger the build